manage.py

```diff
@@ -5,13 +5,22 @@
 from openelevationservice.server.config import SETTINGS
 from openelevationservice.server.db_import.models import db
 from openelevationservice.server.db_import import filestreams
+from openelevationservice.server.grpc.grpc_server import grpc_serve
 
 import click
 
 log = get_logger(__name__)
 
 app = create_app()
 
+@app.cli.command()
+def grpc():
+    """Starts the gRPC server"""
+    grpc_url = '127.0.0.1:5005'
```

Comment:
@carloshellin The url of grpc should be stored in an environment variable?

Reply:
In this case, the gRPC server, just like the HTTP server of OES, will always listen on 127.0.0.1 regardless of the environment. It is the other microservices that connect to this service that need to change the way they connect. And the port can always be changed in the docker-compose. Therefore, I don't think it's necessary.